src/ontology/mondo-edit.obo

```diff
@@ -464918,6 +464920,2857 @@ is_a: MONDO:0005550 {source="https://orcid.org/0000-0002-5002-8648"} ! infectiou
+intersection_of: MONDO:0005583 {source="https://orcid.org/0000-0002-5002-8648"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0004614 {source="https://orcid.org/0000-0002-5002-8648"} ! cross-species analog chronic monocytic leukemia
+
+[Term]
+id: MONDO:1010214
+name: non-hodgkin lymphoma, non-human animal
+def: "Non-hodgkin lymphoma that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:001708 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:001708"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="https://orcid.org/0000-0002-5002-8648"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0018908 {source="https://orcid.org/0000-0002-5002-8648"} ! cross-species analog non-Hodgkin lymphoma
+
+[Term]
+id: MONDO:1010215
+name: familial adenomatous polyposis, non-human animal
+def: "Familial adenomatous polyposis that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:001916 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:001916"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:001916"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0021056 {source="OMIA:001916"} ! cross-species analog familial adenomatous polyposis 1
+
+[Term]
+id: MONDO:1010216
+name: embryonal rhabdomyosarcoma, non-human animal
+def: "Embryonal rhabdomyosarcoma that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:001921 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:001921"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="https://orcid.org/0000-0002-5002-8648"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0009993 {source="https://orcid.org/0000-0002-5002-8648"} ! cross-species analog embryonal rhabdomyosarcoma
+
+[Term]
+id: MONDO:1010217
+name: schwannomatosis, non-human animal
+def: "Schwannomatosis that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:001963 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:001963"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="https://orcid.org/0000-0002-5002-8648"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0008075 {source="https://orcid.org/0000-0002-5002-8648"} ! cross-species analog schwannomatosis
+
+[Term]
+id: MONDO:1010220
+name: ameloblastoma, non-human animal
+def: "Ameloblastoma that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:002171 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:002171"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="https://orcid.org/0000-0002-5002-8648"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0017795 {source="https://orcid.org/0000-0002-5002-8648"} ! cross-species analog ameloblastoma
+
+[Term]
+id: MONDO:1010221
+name: neurofibromatosis type 1, non-human animal
```

Comment:
In human, "disease ~ type x" is based on the gene affected. Should we have only the species-specific term(s) instead of this generic term based on "general species unspecific" gene? 
Note that these species-specific diseases (based on species-specific genes) could be analogous to the human type x disease, but it is difficult to create an axiom referring to the general "species-not- specific" gene.